data/mods/Xedra_Evolved/recipes/inventor/mics.json

```
@@ -0,0 +1,154 @@
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "light_minus_disposable_cell", 1 ] ],
+      [ [ "power_supply", 1 ] ],
+      [ [ "scrap", 3 ] ],
+      [ [ "cable", 40 ] ],
+      [ [ "plastic_chunk", 10 ] ],
+      [ [ "amplifier", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "inventor_medium_cell",
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_MICS",
+    "skill_used": "deduction",
+    "difficulty": 5,
+    "skills_required": [ "electronics", 3 ],
+    "time": "3 h",
+    "reversible": true,
+    "proficiencies": [ { "proficiency": "prof_plasticworking", "time_multiplier": 2, "skill_penalty": -100 } ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "light_minus_disposable_cell", 1 ] ],
+      [ [ "power_supply", 1 ] ],
+      [ [ "scrap", 3 ] ],
+      [ [ "cable", 60 ] ],
+      [ [ "plastic_chunk", 10 ] ],
+      [ [ "amplifier", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "inventor_heavy_cell",
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_MICS",
+    "skill_used": "deduction",
+    "difficulty": 5,
+    "skills_required": [ "electronics", 3 ],
+    "time": "3 h",
+    "reversible": true,
+    "proficiencies": [ { "proficiency": "prof_plasticworking", "time_multiplier": 2, "skill_penalty": -100 } ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "light_minus_disposable_cell", 1 ] ],
+      [ [ "power_supply", 1 ] ],
+      [ [ "scrap", 3 ] ],
+      [ [ "cable", 80 ] ],
+      [ [ "plastic_chunk", 10 ] ],
+      [ [ "amplifier", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "inventor_welder",
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_MICS",
+    "skill_used": "deduction",
+    "difficulty": 5,
+    "skills_required": [ "electronics", 3 ],
+    "time": "3 h",
+    "reversible": true,
+    "proficiencies": [
+      { "proficiency": "prof_welding_basic", "time_multiplier": 3, "skill_penalty": -100 },
+      { "proficiency": "prof_welding", "time_multiplier": 2, "skill_penalty": -100 }
+    ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
+    "components": [
+      [ [ "power_supply", 6 ] ],
+      [ [ "scrap", 2 ] ],
+      [ [ "cable", 80 ] ],
+      [ [ "plastic_chunk", 5 ] ],
+      [ [ "magnetron", 2 ] ],
+      [ [ "lens_small", 1 ] ],
+      [ [ "amplifier", 1 ] ],
+      [ [ "duct_tape", 60 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "heater_tool",
+    "category": "CC_XEDRA",
+    "subcategory": "CSC_XEDRA_MICS",
```

```suggestion
    "subcategory": "CSC_XEDRA_MICS",
```
Is this supposed to be `CSC_XEDRA_MISC`

---

Yeah, tools don't have their own group yet, so its temporary in mics group